lib/takeaway.rb

```diff
@@ -0,0 +1,90 @@
+require './lib/menu.rb'
+require './lib/dish.rb'
+
+# create list of dishes
+pad_kapow = Dish.new("pad kapow", 50)
```

Comment:
I like having a pre-defined menu in the code, makes it much easier to read and test than mine I think

Reply:
On the other hand it's also not very flexible. I think the ideal with both yours and mine would be to pre-set the menu in a separate file and then require this file at the top of the main code. This would in theory allow any restaurant to use the same code and load their menu easily. I started to have a look but had difficulty importing variables from an external file.